.github/workflows/tox.yml

```diff
@@ -30,6 +30,10 @@ jobs:
         pip install tox tox-gh-actions
     - name: Test with tox
       run: tox
+    - name: Upload to coverage
+      uses: codecov/codecov-action@v1.0.12
+      with:
+        flags: ${{ matrix.python-version }}
```

Comment:
I don't know if it works or not now. Just in case, I fixed that problem elsewhere by not using the codecov action but directly use a bash call:
```suggestion
      bash <(curl -s https://codecov.io/bash)
```

Reply:
Done and rebased (and also provided the python version as flags via the environment)